.yamllint

```diff
@@ -7,11 +7,12 @@ ignore:
   - 'k3d/local/manifests/metallb/metallb-native.yaml'
   - '**/.terraform/**'
   - '**/chart/templates**'
+  - '!**/chart/templates/uds-*.yaml'
```

Comment:
Seems like this is more pain than its worth, so I think we can just drop it. Might want to evaluate helm lint smoke tests in CI to handle the charts (could also do `helm template` -> `yamllint`). Regardless - out of the scope of your PR so can probably just revert the changes here for now.

Reply:
Opened https://github.com/defenseunicorns/uds-core/issues/231 as a follow on to this.